sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -1121,6 +1121,29 @@ abstract class ParquetFilterSuite extends QueryTest with ParquetTest with Shared
         }
       }
     }
+
+    withSQLConf(
+      // Makes sure disabling 'spark.sql.parquet.recordFilter' still enables
+      // row group level filtering.
+      SQLConf.PARQUET_RECORD_FILTER_ENABLED.key -> "false",
+      SQLConf.PARQUET_FILTER_PUSHDOWN_ENABLED.key -> "true",
+      SQLConf.PARQUET_VECTORIZED_READER_ENABLED.key -> "false") {
```

Comment:
Can we test both vectorized and non-vectorized reader?

Reply:
+1, we can merge this to be code block above, which is inside a `Seq(true, false).foreach { vectorized =>`

Reply:
Done. Thanks.